Sender/fileReader.cpp

```diff
@@ -0,0 +1,32 @@
+#include"fileReader.h"
+
+FileReader::FileReader(std::string filePathName) :_fileName("./Sender/SafeFile.csv"), ReadIsGood(false)
+{
+	std::ifstream tempPtr;
+	tempPtr.open(filePathName);
+	if (tempPtr.good())
+		_fileName = filePathName;
+
+	tempPtr.close();
+
+	ReadIsGood = ReadFile();
+}
+
+bool FileReader::ReadFile()
+{
+	_filePtr.open(_fileName);
```

Comment:
good to enclose file IO in a try, catch block.

Reply:
done